app/src/main/java/chat/rocket/android/helper/MessageParser.kt

```diff
@@ -70,7 +70,8 @@ class MessageParser @Inject constructor(
 
         message.mentions?.forEach {
             val mention = getMention(it)
-            mentions.add(mention)
+            if(mention.isNotEmpty())
```

Comment:
There is a missing space here after `if`. It would also be better if you can add curly brackets for that if statement. :smiley: